### DISCUSSION TITLE
[Unknown Mod][Adventure Map] Replace the text "(already has max skills)" by the text "(full skills)" for Witch Hut

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#3305] [Ideas]
Remplace the text "(already has max skills)" by the text "(full skills)" for Witch Hut.

In fheroes2:

![No room for skill 01](https://user-images.githubusercontent.com/43583994/116488338-736ac200-a892-11eb-8d67-2f556e57c185.png)

This is not an option.
This can be implemented directly in fheroes2.

**Comment by Branikolog** [comment]
Hi, @LeHerosInconnu 
I think the phrase "All skills learned" could be considered a hero to have all possible skills, which is not the case...
Anyway, I don't like current text be placed on two lines too...

**Reply by LeHerosInconnu** [reply]
Hello @Branikolog,

In this case, let's use "(skill slots full)".

**Comment by LeHerosInconnu** [comment]
Let's use "(full skills)".
